src/Sylius/Bundle/CoreBundle/Resources/config/form.xml

```diff
@@ -48,6 +48,8 @@
         <parameter key="sylius.form.type.promotion_rule.customer_group_configuration.class">Sylius\Bundle\CoreBundle\Form\Type\Rule\CustomerGroupType</parameter>
         <parameter key="sylius.form.type.promotion_action.shipping_discount_configuration.class">Sylius\Bundle\CoreBundle\Form\Type\Action\ShippingDiscountConfigurationType</parameter>
         <parameter key="sylius.form.type.promotion_action.add_product_configuration.class">Sylius\Bundle\CoreBundle\Form\Type\Action\AddProductConfigurationType</parameter>
+
+        <parameter key="sylius.form.type.product_review_admin.class">Sylius\Bundle\CoreBundle\Form\Type\ProductReviewAdminType</parameter>
```

Comment:
Why do we need that here? It should be customized in configuration if core requires that.